src/System.Memory.Polyfill/System/Parse.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    public static partial class Int32Polyfill
+    {
+        public static bool TryParse(ReadOnlySpan<char> buffer, out int value)
```

Comment:
Would,'t work for implicit cast from `Span<char>`?

Reply:
Why not?  It's not an extension method.

Reply:
I am not sure I understand. Can you clarify?